redis-formula/pillar.example

```diff
@@ -0,0 +1,21 @@
+redis:
+
+  # this example will instantiate a "redis@myapp" service
+  myapp:
+    # any available Redis settings can be defined
+    port: 0
+
+    # the following parameters will be set automatically unless they are overwritten
+    timeout: 0
+    supervised: systemd
+    unixsocket: /run/redis/myapp.sock
+    unixsocketperm: 460
```

Comment:
`460` permissions are quite unusual. Does redis really need these special socket permissions, or did you mean `640`?

Reply:
It needs to be group-writeable to allow other applications to use it. The owner permissions are not so relevant, I can make it `660` to make it more "usual", but I don't think it would change the functionality.

Reply:
Then let's keep the lower permissions to give the impression of more security. ;-)